CONTRIBUTING.md

```diff
@@ -30,7 +30,30 @@ Please also give the code of conduct a read.
 
 ## How do I set up the project?
 
-Run `npm install` and edit code in the `src/` folder. It's luckily very simple! :wink:
```

Comment:
We shouldn't remove this :wink:

Instead can we add another title: "How do I verify and test my changes?".
There we can not only talk about the sandbox, but also about the tests